exercises/gigasecond/introduction.md

```diff
@@ -0,0 +1,20 @@
+# Introduction
+
+The way we measure time is kind of messy.
+We have 60 seconds in a minute, and 60 minutes in an hour.
+This comes from ancient Babylon, where they used 60 as the basis for their number system.
+We have 24 hours in a day, 7 days in a week, and months?
+Well, for months it depends not only on which month it is, but also what calendar is used in the country you live in.
+
+If we ever colonize Mars or some other planet, it's going to get even messier.
+If someone says "year" do they mean Earth-year or Mars-year?
+
+Instead, we could use the metric system, and count everything in seconds.
```

Comment:
Are you open to subheadings? The story is taking a turn at this point.
```suggestion
## What if we only used seconds?

We can express time intervals just with seconds.
Then we can use metric system prefixes for writing large numbers of seconds in more easily comprehensible quantities.
```

Reply:
My gut instinct says that if we open up for subheadings we're encouraging _more_, where I am subtly hoping for less.

I like the way you broke this up, though, so I think I want to build on that. (I'm open to changing my mind on the subheading thing, I just don't have a good sense of the arguments yet).